app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/components/themed/MediaItemMenu.kt

```diff
@@ -93,7 +93,7 @@ fun InHistoryMediaItemMenu(
                 query {
                     // Not sure we can to this here
                     context.globalCache.removeResource(song.id)
-                    Database.unMarkDownloaded(song.id)
+                    Database.markDownloaded(song.id, false)
```

Comment:
is it the only place where we delete the song? I suppose we have something in settings->cache too. Please check.
Other than that - looking good to me!

Reply:
I can't find other places where we can delete songs.

Reply:
ok neither can I